network/router.go

```diff
@@ -22,7 +22,7 @@ import (
 //   router.Stop() // will stop the listening and the managing of all Conn
 type Router struct {
 	// id is our own ServerIdentity
-	id *ServerIdentity
+	ServerIdentity *ServerIdentity
```

Comment:
I was only referring to the renaming. It's good to do such refactoring if you see it, just open a separate PR so it's easier to review the changes here.